extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -348,9 +362,7 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       event.stopPropagation();
     });
 
-    this.vsync_.mutate(() => {
-      this.container_.appendChild(this.descriptionBox_);
-    });
+    this.controlsContainer_.appendChild(this.descriptionBox_);
```

Comment:
why does this call also not go inside the vsync?

Reply:
Oh! It's because the `controlsContainer_` isn't attached to the dom. Basically, all of the attachments to `controlsContainer_` are created and attached offline, and the final attachment of `controlsContainer_` to `container_` is put inside the vsync mutate. XD

Reply:
Ahh that makes sense. Thank you for the clarification! :) One follow up, the call to attaching `controlsContainer_ ` to `container_ ` is inside `buildControl_()` however, that call isn't inside a vsync mutate call either?

Reply:
Good catch!